wiki/en/Running-a-Server.md

```diff
@@ -150,21 +141,17 @@ Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s
 
 <figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus server setup window"></figure>
 
-### Make My Server Public
-
-By default, you will be running a Private Server and need to [read these instructions](#running-a-private-server) to have others connect to you in this mode.
+### The Directory list 
 
-When making your server public, you should see a confirmation message saying whether your server has registered successfully. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.
+**None**: By default, you will not be connected to a directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your server in this mode.
 
+**Genre**: To allow other people to see your server on one of the built-in public directories, select your desired genre directory. You should see a confirmation message saying whether your server has registered successfully. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.  
 
-### Genre
-
-Because there is a limit of 150 servers per Directory, you need to select which Directory Server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). This will make your server appear in the relevant server list for clients.
-
+**Custom**: This sets your server to be a directory on which other servers can be listed. Note that when setting this mode, you may want also to specify the location of server list file (See "Options" below) unless you want to disconnect all listed servers when restarting Jamulus.
```

Comment:
> This sets your server to be a directory on which other servers can be listed.

As noted else where, this is wrong.

Custom Directories are where Servers register so they can be seen by Clients viewing that Customer Directory in the Connect window.

Setting the select box to "Custom" and setting a Custom Directory server address is how to do this.

For the tiny number of operators running Directories, putting "localhost" into the Custom Directory server address means you're running as a Custom Directory yourself, rather than registering your server with a Custom Directory.